src/parameters/create_parameters.jl

```diff
@@ -4,54 +4,48 @@ import CLIMAParameters as CP
 import RRTMGP.Parameters as RP
 import SurfaceFluxes as SF
 import SurfaceFluxes.UniversalFunctions as UF
-import ClimaCore
-import ClimaCore as CC
 import Insolation.Parameters as IP
 import Thermodynamics as TD
 import CloudMicrophysics as CM
+# TODO: Remove these imports?
+import ClimaCore
+import ClimaCore as CC
 
-function override_climaatmos_defaults(
-    defaults::NamedTuple,
-    overrides::NamedTuple,
-)
-    intersect_keys = intersect(keys(defaults), keys(overrides))
-    intersect_vals = getproperty.(Ref(overrides), intersect_keys)
-    intersect_overrides = (; zip(intersect_keys, intersect_vals)...)
-    return merge(defaults, intersect_overrides)
-end
+function create_parameter_set(config::AtmosConfig)
+    # Helper function that creates a parameter struct. If a struct has nested 
+    # parameter structs, they must be passed to subparam_structs as a NamedTuple.
+    function create_parameter_struct(param_struct; subparam_structs = (;))
```

Comment:
Does this need to be defined in `create_parameter_set`?

Reply:
It doesn't need to be, but it makes the syntax nicer since it automatically grabs the correct `toml_dict` and `parsed_args` without them needing to be passed in.